src/main/java/com/internet/shop/controllers/InjectDataController.java

```diff
@@ -24,12 +26,11 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        User userBob = new User("Bob");
-        User userTom = new User("Tom");
-        userBob.setPassword("12345");
-        userTom.setPassword("12345");
-        userService.create(userBob);
-        userService.create(userTom);
+        User admin = new User("admin", "admin");
+        Role role = new Role(Role.RoleName.ADMIN);
```

Comment:
This is optional, but instead of creating a new object, since you already have a list of roles, you can simply set this role to the user using for example:
userService.create(new User("Admin", "Admin", "1", Set.of(Role.of("ADMIN"))));